src/pfs/pfs.proto

```diff
@@ -339,11 +295,13 @@ message CopyFile {
 }
 
 message ModifyFileRequest {
-  Commit commit = 1;
-  oneof modification {
-    PutFile put_file = 2;
+  oneof body {
+    Commit set_commit = 1;
+    AddFile add_file = 2;
     DeleteFile delete_file = 3;
     CopyFile copy_file = 4;
+
+    google.protobuf.BytesValue tar_data = 5; 
```

Comment:
Lets just get rid of this. It doesn't make sense in this model and I don't think it is too much of a lift to handle this in the client (going from tar to add files).